src/internals/terms.md

```diff
@@ -54,11 +75,13 @@ based on those ranges is defined as follows,
 
 \\[\begin{eqnarray}
 [r_1] \land [r_2] &=& [r_1 \cap r_2],                 \nonumber \\\\
-[r_1] \land \neg [r_2] &=& [r_1 \cap \overline{r_2}], \nonumber \\\\
+[r_1] \land \neg [r_2] &=& [r_1 \cap complement(r_2)], \nonumber \\\\
```

Comment:
Is that intentionally the text and not a symbol?

Reply:
Few sentences before, I'm saying that I'd be using overline for terms, and here I use it on a range set, which felt a bit inconsistent. Not sure how to proceed with this one.

Reply:
Maybe the upper C as showed here would be better https://en.wikipedia.org/wiki/Complement_(set_theory) ?

Reply:
![image](https://github.com/pubgrub-rs/guide/assets/2905865/7e0f2c42-d215-4484-872f-1ae882262e41)

Is that better? https://github.com/pubgrub-rs/guide/pull/11/commits/37ba8c355db366514a4f0b7ed9533e82022c7bb5

Reply:
I like it